### PR TITLE
3.13 oc 7567

### PR DESCRIPTION
**Pull request by hieudd**
Fix study attr in /RestoreStudyEvent & /RemoveStudyEvent
(cherry picked from commit 3031028)

### CODE REVIEW

**Comment by sdibona** [comment]
Hello,

Thank you for re-creating this pull request against the 3.13 release.  I am reviewing the code changes and will provide an update tomorrow with any suggestions or comments.

Thank you,
-Steve DiBona

**Comment by sdibona** [comment]
Hello,

Code look good.  A couple minor comments added to the changes.

I wanted to let you know that due to a tight release deadline there is not enough time to adequately test this change (an automated test needs to be created) before we need to be ready for release.  So we will, unfortunately, not be able to include it as part of 3.13.  However we are very much planning on including it in an upcoming release.  I will provide further updates when more information becomes available.

Please let me know if you have any questions.

Thanks again,
Steve DiBona

File: web/src/main/java/org/akaza/openclinica/control/managestudy/RemoveStudyEventServlet.java [hunk 1]
```diff
@@ -97,7 +97,8 @@ public void processRequest() throws Exception {
 
             StudyDAO studydao = new StudyDAO(sm.getDataSource());
             StudyBean study = (StudyBean) studydao.findByPK(studySub.getStudyId());
-            request.setAttribute("study", study);
+            // @sinzii: changed attr name from study to subStudy
```
Comment (sdibona) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/RemoveStudyEventServlet.java]: Please remove this comment.  It is not explaining functionality and is not needed.

File: web/src/main/java/org/akaza/openclinica/control/managestudy/RestoreStudyEventServlet.java [hunk 2]
```diff
@@ -104,7 +104,8 @@ public void processRequest() throws Exception {
 
             StudyDAO studydao = new StudyDAO(sm.getDataSource());
             StudyBean study = (StudyBean) studydao.findByPK(studySub.getStudyId());
-            request.setAttribute("study", study);
+//            request.setAttribute("study", study);
```
Comment (sdibona) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/RestoreStudyEventServlet.java]: Please remove this comment.  We avoid committing commented out code.

**Comment by hieudd** [comment]
Thank Steve for your response.

**Comment by sdibona** [comment]
Changes look good. Merging into 3.14.

**Pull request merged**